core/src/scene/sceneLoader.cpp

```diff
@@ -722,12 +722,25 @@ void SceneLoader::loadCameras(Node _cameras, Scene& _scene) {
 
             view->setCameraType(CameraType::perspective);
 
-            if (Node fov = camera["fov"]) {
-                // TODO
-            }
-
+            // Only one of focal length and FOV is applied;
+            // according to docs, focal length takes precedence.
             if (Node focal = camera["focal_length"]) {
-                // TODO
+                if (focal.IsScalar()) {
+                    float length = focal.as<float>(view->getFocalLength());
```

Comment:
Also arguments for `as<>(const S& fallback)` are fallbacks if yaml node for which `as` is called is not valid.

In this case, since we are calling `focal.IsScalar()` on it, it will always be valid.